pip-dep/requirements.txt

```diff
@@ -9,6 +9,7 @@ requests~=2.22.0
 scipy~=1.4.1
 syft-proto~=0.2.9.a1
 tblib~=1.6.0
+tenseal
```

Comment:
This dependency should specify a version:
```suggestion
tenseal~=0.1.0.dev0
```

Reply:
done